runtime/types.go

```diff
@@ -72,6 +72,32 @@ type Options interface {
 	// limit is primarily offered to defend against unintentional bursts of new
 	// time series being inserted.
 	WriteNewSeriesLimitPerShardPerSecond() int
+
+	// SetTickPerSeriesSleepDuration sets the tick sleep per series value that
+	// provides a constant duration to sleep per series at the end of processing
+	// a batch of series during a background tick, this can directly effect how
```

Comment:
"batch of series" is a little ambigous. This sleep is after processing a single series isn't it?

Reply:
Its after a group, not at the end of each one, primarily to go easy on the scheduler.  A sleep at the end of each one would be a lot more yielding/continuation than doing it in groups of 4096 (current value).